app/views/hyrax/collections/_show_parent_collections.html.erb

```diff
@@ -0,0 +1,30 @@
+<% if @parent_collections.nil? || @parent_collections.empty? %>
+		<div class="alert alert-warning" role="alert"><%= t('hyrax.collections.show.no_visible_parent_collections') %></div>
+<% else %>
+		<div class="media-body">
+			<h4 class="media-heading">
```

Comment:
Typically you would't want to keep content within a `<h4>` or any headline `<hX>` tag just text, and not embed markup (like the nested `<ul><li>...`).  If it works that's fine, but will raise some accessibility flags.

Reply:
Yea, I wouldn't normally do that.  I copied it from _subcollection_list.html.erb and didn't update it.  I think that was being used to make the collection names bold and larger.  That pattern is used in a number of places.

I think I will leave it for now and we should probably revisit with a refactor of all the places it is used.

Reply:
Created issue #2505 to request this pattern to be refactored.